doc/deployment_guide.md

```diff
@@ -0,0 +1,5450 @@
+
+<img src="media/dc99730255437cdd115b74c45cc6c19d.jpg" alt="drawing" width="200px"/>
+
+# Dell EMC Red Hat Ready Architecture Deployment Guide Notes

```

Comment:
I would remove Notes from the end of the title.

Reply:
removed